.gitignore

```diff
@@ -10,3 +11,4 @@ env/
 htmlcov/
 nosetests.xml
 octodns_*.egg-info/
+pytest.ini
```

Comment:
What's the origin story of this one? Something vscode created? If so it's it's probably a global ignores thing. If it was manually created then probably just shouldn't be here, any pytest config should be in pyproject.toml. If it was created by something else then not sure.

Reply:
I added this to help VS Code find tests before the pytest config was moved to pyproject.toml. Not needed now, I'll remove it.